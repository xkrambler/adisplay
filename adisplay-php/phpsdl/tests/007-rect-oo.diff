036+ NULL
037+ +++ Inter (false)
038+ bool(false)
039+ bool(false)
040+ NULL
041+ +++ Uninon
042+ NULL
043+ NULL
044+ NULL
045+ NULL
046+ +++ InterLine
047+ bool(true)
048+ int(0)
049+ int(10)
050+ int(40)
051+ int(30)
036- object(SDL_Rect)#6 (4) {
037-   ["x"]=>
038-   int(16)
039-   ["y"]=>
040-   int(17)
041-   ["w"]=>
042-   int(14)
043-   ["h"]=>
044-   int(13)
045- }
046- +++ Inter (false)
048- bool(false)
053+ Done
049- NULL
050- +++ Uninon
051- NULL
052- object(SDL_Rect)#6 (4) {
053-   ["x"]=>
054-   int(10)
055-   ["y"]=>
056-   int(10)
057-   ["w"]=>
058-   int(28)
059-   ["h"]=>
060-   int(31)
061- }
062- NULL
063- object(SDL_Rect)#6 (4) {
064-   ["x"]=>
065-   int(10)
066-   ["y"]=>
067-   int(10)
068-   ["w"]=>
069-   int(110)
070-   ["h"]=>
071-   int(20)
072- }
073- +++ InterLine
074- bool(true)
075- int(10)
076- int(15)
077- int(29)
078- int(24)
079- bool(false)
080- Done