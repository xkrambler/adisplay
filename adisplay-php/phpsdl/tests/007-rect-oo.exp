+++ Empty (true)
object(SDL_Rect)#1 (4) {
  ["x"]=>
  int(0)
  ["y"]=>
  int(0)
  ["w"]=>
  int(0)
  ["h"]=>
  int(0)
}
string(17) "SDL_Rect(0,0,0,0)"
bool(true)
+++ Empty (false)
object(SDL_Rect)#2 (4) {
  ["x"]=>
  int(1)
  ["y"]=>
  int(2)
  ["w"]=>
  int(3)
  ["h"]=>
  int(4)
}
string(17) "SDL_Rect(1,2,3,4)"
bool(false)
+++ Equal (true)
bool(true)
bool(true)
+++ Equal (false)
bool(false)
bool(false)
+++ Inter (true)
bool(true)
bool(true)
object(SDL_Rect)#6 (4) {
  ["x"]=>
  int(16)
  ["y"]=>
  int(17)
  ["w"]=>
  int(14)
  ["h"]=>
  int(13)
}
+++ Inter (false)
bool(false)
bool(false)
NULL
+++ Uninon
NULL
object(SDL_Rect)#6 (4) {
  ["x"]=>
  int(10)
  ["y"]=>
  int(10)
  ["w"]=>
  int(28)
  ["h"]=>
  int(31)
}
NULL
object(SDL_Rect)#6 (4) {
  ["x"]=>
  int(10)
  ["y"]=>
  int(10)
  ["w"]=>
  int(110)
  ["h"]=>
  int(20)
}
+++ InterLine
bool(true)
int(10)
int(15)
int(29)
int(24)
bool(false)
Done