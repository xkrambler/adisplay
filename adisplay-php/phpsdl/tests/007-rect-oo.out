+++ Empty (true)
object(SDL_Rect)#1 (4) {
  ["x"]=>
  int(0)
  ["y"]=>
  int(0)
  ["w"]=>
  int(0)
  ["h"]=>
  int(0)
}
string(17) "SDL_Rect(0,0,0,0)"
bool(true)
+++ Empty (false)
object(SDL_Rect)#2 (4) {
  ["x"]=>
  int(1)
  ["y"]=>
  int(2)
  ["w"]=>
  int(3)
  ["h"]=>
  int(4)
}
string(17) "SDL_Rect(1,2,3,4)"
bool(false)
+++ Equal (true)
bool(true)
bool(true)
+++ Equal (false)
bool(false)
bool(false)
+++ Inter (true)
bool(true)
bool(true)
NULL
+++ Inter (false)
bool(false)
bool(false)
NULL
+++ Uninon
NULL
NULL
NULL
NULL
+++ InterLine
bool(true)
int(0)
int(10)
int(40)
int(30)
bool(false)
Done